server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -136,20 +139,20 @@ module.exports = class ReportUploadRequestHandler {
         });
         return rl;
       }
-    } catch (e) {
+    } catch (e: any) {
       console.log(e.stack);
     }
     return null;
   }
 
-  async loadReportFromBuffer() {
+  async loadReportFromBuffer(): Promise<Interface> {
     return readline.createInterface({
       input: streamline.createReadStream(this.req.files.report[0].buffer),
       crlfDelay: Infinity,
     });
   }
 
-  async loadProductInfo() {
+  async loadProductInfo(): Promise<any> {
```

Comment:
I'm not 100% sure but I think that product info has defined output type, we could use it instead of any.

Reply:
I've made Type base on the schema